app/mailers/comment_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CommentMailer < ApplicationMailer
```

Comment:
Since "comment emails" are not related to the StackExchange integration, it would help if this code was in a separate pull request. Small pull requests are best!